test/unit/test_events.py

```diff
@@ -22,6 +21,8 @@
 def get_all_subclasses(cls):
     all_subclasses = []
     for subclass in cls.__subclasses__():
+        if subclass in [NodeInfo, AdapterEventBase, TestLevel, DebugLevel, WarnLevel, InfoLevel, ErrorLevel]:
```

Comment:
lets add a comment here that if the test breaks because of abcs that this list might have to be updated

Reply:
ok. Done.